spark-and-cassandra-on-centos/azuredeploy.json

```diff
@@ -0,0 +1,790 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageMasterType": {
+      "metadata": {
+        "description": "Storage type that is used for master Spark node.  This storage account is used to store VM disks."
+      },
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_ZRS",
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Premium_LRS"
+      ]
+    },
+    "storageSlaveType": {
+      "metadata": {
+        "description": "Storage type that is used for each of the slave Spark node.  This storage account is used to store VM disks."
+      },
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_ZRS",
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Premium_LRS"
+      ]
+    },
+    "storageCassandraType": {
+      "metadata": {
+        "description": "Storage type that is used for Cassandra.  This storage account is used to store VM disks."
+      },
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_ZRS",
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Premium_LRS"
+      ]
+    },
+    "vmMasterVMSize": {
+      "metadata": {
+        "description": "VM size for master Spark node.  This VM can be sized smaller."
+      },
+      "type": "string",
+      "defaultValue": "Standard_D1_v2",
+      "allowedValues": [
```

Comment:
That makes more sense.  I'll add a few common SKUs since not all of the SKUs are available across the regions.